fairlearn/metrics/__init__.py

```diff
@@ -23,6 +23,8 @@
 for each combination of subgroups.
 """
 
+from ._metric_frame import MetricFrame  # noqa: F401
+from ._make_derived_metric import make_derived_metric  # noqa: F401
 
 from ._metrics_engine import _derived_metric_dict
```

Comment:
This would need to be migrated to the new `make_derived_metric()` before checkin.